### ISSUE TITLE
Show more than one categorie

### ISSUE THREAD

**Issue by globinli** [joomla-extensions/weblinks#267]
#### Steps to reproduce the issue
At the moment, I can display different categories, which are first to open oder I display the links of one categorie

#### Expected result
For a page with diffent kind of links it would be useful to display all categories with all links shown

#### System information (as much as possible)
Joomla 3.4, Weblinks 3.5

**Comment by Bakual** [comment]
This could be done in a smilar way like we do in com_content, optionally showing items from subcategories (see https://github.com/joomla/joomla-cms/blob/staging/components/com_content/models/articles.php#L355-L381)

**Comment by johannydls** [comment]
Why do not you create a way to show all items in all categories at once? For example, I could have a section of links, where they would be divided into 4 sections, cat1, cat2, cat3 and cat4, but I would also like to show all links at once, being possible to sort them. Is it possible to do this in the current version?

**Comment by franz-wohlkoenig** [comment]
I agree on Possibility to show in iE. Top-Category all Links and also Sub-Categories.
